exampleSite/config/_default/params.toml

```
@@ -127,6 +127,7 @@ commento = false
 [utterances]       # https://utteranc.es/
   owner = ""              # Your GitHub ID
   repo = ""               # The repo to store comments
+  issueTerm =""            # Choose the mapping between blog posts and GitHub issues. pathname (default), title, url, og:title...
```

```suggestion
  issueTerm = ""            # Choose the mapping between blog posts and GitHub issues. pathname (default), title, url, og:title...
```

Nitpicking, but it would be "nicer" to follow the formatting convention (used all around)